android/src/main/java/com/twilio/verify/reactnative/RNTwilioVerifyModule.kt

```diff
@@ -146,6 +147,7 @@ class RNTwilioVerifyModule(
             { promise.resolve(null) },
             { exception -> promise.reject(exception) })
         }
+      else -> promise.reject(IllegalArgumentException("Invalid update challenge payload"))
```

Comment:
Nice to have: locate the error message , I'd suggest a Custom Exception and let the consumer implement the exception catching